logger.go

```diff
@@ -72,3 +73,18 @@ func NewServiceLogger(serviceName string, level int) ServiceLogger {
 	}
 	return logger
 }
+
+// GetLogger initializes a logging object writing to the requested log file.
+func GetLogger(logFile string, serviceName string) *log.Logger {
```

Comment:
this should accept the interface it cares about (`io.writer`), not a magic string that is impartially checked for usefulness